content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md

```diff
@@ -1,32 +1,43 @@
+- **Ready-to-Use Playbooks**: a range of pre-built playbooks optimized for common security workflows, allowing teams to quickly deploy effective responses with minimal configuration.
+
+The playbooks included in this pack can help you save time and keep your incidents and indicators in sync with Cortex XDR. They also help automate repetitive tasks:
 
-The playbooks included in this pack help you save time and keep your incidents in sync with Cortex XDR. They also help automate repetitive tasks associated with Cortex XDR incidents:
 - Syncs and updates Cortex XDR incidents.
 - Triggers a sub-playbook to handle each alert by type.
 - Extracts and enriches all relevant indicators from the source alert.
 - Hunts for related IOCs.
-- Calculates the severity of the incident.
-- Interacts with the analyst to choose a remediation path or close the incident as a false positive based on the gathered information and incident severity.
-- Remediates the incident by blocking malicious indicators and isolating infected endpoints.
+- Assesses the severity of the incident.
+- Collaborates with the analyst to determine the appropriate remediation path or classify the incident as a false positive, based on gathered data and incident severity.
+- Remediates the incident by blocking malicious indicators and isolating compromised endpoints.
+
+## The Palo Alto Networks Cortex XDR - Investigation and Response pack enables the following flows:
 
-The Palo Alto Networks Cortex XDR - Investigation and Response pack enables the following flows: 
 - [Lite Incident Handling](#lite-incident-handling) - A lite playbook for handling Palo Alto Networks Cortex XDR incidents, which encompasses incident enrichment, investigation, and response for each incident.
 - [Device Control Violations](#device-control-violations) - Fetch device control violations from XDR and communicate with the user to determine the reason the device was connected.
-- [XDR Incident Handling](#xdr-incident-handling) - Compare incidents in Palo Alto Networks Cortex XDR and Cortex XSOAR, and update the incidents appropriately. 
-- [Cloud IAM User Access Investigation](#cloud-iam-user-access-investigation) - Investigates and responds to Cortex XDR Cloud alerts where an Cloud IAM user's access key is used suspiciously to access the cloud environment. 
-- [Cortex XDR - Cloud Cryptomining](#Cortex_XDR_-_Cloud_Cryptomining) - Investigates and responds to Cortex XDR XCloud 
+- [XDR Incident Handling](#xdr-incident-handling) - Compare incidents in Palo Alto Networks Cortex XDR and Cortex XSOAR, and update the incidents appropriately.
+- [Cloud IAM User Access Investigation](#cloud-iam-user-access-investigation) - Investigates and responds to Cortex XDR Cloud alerts where an Cloud IAM user's access key is used suspiciously to access the cloud environment.
+- [Cortex XDR - Cloud Cryptomining](#Cortex-XDR-Cloud-Cryptomining) - Investigates and responds to Cortex XDR XCloud
```

Comment:
should this be "Investigates and responds to Cortex XDR Cloud" or "Investigates and responds to Cortex XDR XCloud"